Samples/tutorials/first-run-experience-tutorial/README.md

```diff
@@ -0,0 +1,106 @@
+---
+page_type: sample
+urlFragment: office-add-in-first-run-experience-tutorial
+products:
+  - office
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: 09/05/2024 4:00:00 PM
+description: "A completed version of the step-by-step first-run experience tutorial hosted on learn.microsoft.com."
+---
+
+# Office Add-in First-run Experience Tutorial - Completed
+
+## Summary
+
+This sample is the result of completing the [Build an Office Add-in with a basic first-run experience](https://learn.microsoft.com/office/dev/add-ins/tutorials/first-run-experience-tutorial). It was constructed with the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
```

Comment:
Consider removing the only past tense in the readme. Also, I think the first sentence makes more sense when you explicitly label the article as a "tutorial".

```suggestion
This sample is the result of completing the [Build an Office Add-in with a basic first-run experience](https://learn.microsoft.com/office/dev/add-ins/tutorials/first-run-experience-tutorial) tutorial. It is built from the [Yeoman generator for Office Add-ins](https://learn.microsoft.com/office/dev/add-ins/develop/yeoman-generator-overview).
```